config/locales/en.yml

```diff
@@ -23,6 +23,7 @@ en:
           nostdin: Do not read from stdin
           out_format: 'Output format: %{valid_text}'
           pattern_router: 'Use pattern router: keys moved per config data.write'
+          skip_interpolation: Skip translating strings that are straight interpolations (eg. "%{comment}")
```

Comment:
Rather than adding an option, perhaps we should always simply copy such keys verbatim? There is never a reason to translate them.

Reply:
That would be preferential in our use case but I didn't want to assume...

Reply:
Actually we don't even copy them, we have Rails configured with fallback which then just works... Not sure that's a safe assumption either though.

Reply:
There is never any reason to translate interpolation-only values, so let's skip them unconditionally.

Reply:
Sounds good to me! :)

Reply:
Sure, makes sense to have that as an option.